DotNetRu.Android/DotNetRu.Android.csproj

```diff
@@ -15,7 +15,7 @@
     <AndroidResgenClass>Resource</AndroidResgenClass>
     <AndroidResgenFile>Resources\Resource.designer.cs</AndroidResgenFile>
     <AndroidApplication>True</AndroidApplication>
-    <AndroidUseLatestPlatformSdk>true</AndroidUseLatestPlatformSdk>
+    <AndroidUseLatestPlatformSdk>false</AndroidUseLatestPlatformSdk>
```

Comment:
Why not use latest SDK?

Reply:
It's recommended to set compile version equal to target
It's guaranteed that all will build project using Android 8 => less problems (we all know ~~stability~~ of Xamarin)